### DISCUSSION TITLE
polygon

### DISCUSSION THREAD

**Discussion by OneLoneCoder** [OneLoneCoder/olcUTIL_Geometry2D#59] [General]
Just wanted to kick off polygon discussions here

**Comment by JuliusDiestra** [comment]
Hello !
I am interested on this struct implementation. I saw there is a PR already open, but asking just in case.
which methods would you like to add in this struct?
side_count()
side(.)
perimeter()
area() 
I am not sure if you are interested to add a methods to : 
1) Check if Polygon is regular
2) Check if Polygon is convex
3) Do Polygon triangulation using vertex information.

Greetings !
